lib/queue_kit/signal_checker.rb

```diff
@@ -36,7 +36,6 @@ def trap_signal(sig)
       debug :setup, sig
 
       old_handler = trap sig do
-        debug :trap, sig
```

Comment:
Dug into it more: it calls the `debug` method which itself is used only to perform instrumentation of the signal being trapped (which obviously doesn't work given previously discussed reasons).